src/models/TranscriptJson.ts

```diff
@@ -33,6 +34,8 @@ export default class TranscriptJson implements Model {
   annotations?: any;
```

Comment:
TranscriptJson is not a database collection, it is just a JSON of the transcript data. https://github.com/CouncilDataProject/cdp-backend/blob/main/cdp_backend/pipeline/transcript_model.py

So, it would not have an `id`. `confidence`, `generator`, `created_datetime`, `sentences` are required fields.

Reply:
got it, I was wondering what they were doing in the models because I couldn't find it in the [database schema](https://councildataproject.org/cdp-backend/database_schema.html)